src/xrCore/_flags.h

```diff
@@ -13,17 +13,20 @@ struct _flags
 
     T flags;
 
+    _flags () { flags = static_cast<T>(0); }
+    _flags (T v) { flags = static_cast<T>(v); }
+
     TYPE get() const noexcept { return flags; }
 
     SelfRef zero() noexcept
     {
-        flags = T(0);
+        flags = static_cast<T>(0);
         return *this;
     }
 
     SelfRef one() noexcept
     {
-        flags = T(-1);
+        flags = static_cast<T>(~0);
```

Comment:
```suggestion
         flags = static_cast<T>(-1);
```

Reply:
1 is inverted 0, but not -1. -1 it is a signed value.

Reply:
```
#include <stdio.h>
#include <stdlib.h>
int main()
{
    printf("%lx %x %x %x\n", ~0UL, ~0U, (unsigned short)(~0U), (unsigned char)(~0U));
}
```

Reply:
`~0` to `-1` will not change, because `-1` instead of `~0` is a "hack", so people write when they want to show how smart they are, that they say that in the additional code `-1` these are all ones in binary form. But the readability of the code at the same time falls.